src/Pages/cecilia/Main/Comment.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./Comment.scss"
+
+class Comment extends React.Component {
+  // constructor() {
+  //   super();
+  //   this.state = {
+
+  //   }
+  // }
+
+  //fetch함수 사용해서 데이터 가져오기. 아직 생김새가 익숙하진 않지만 눈과 손에 익히기. 
+  // componentDidMount() {
+  //   fetch('http://localhost:3000/data/data.json', {
+  //     method: 'GET'
+  //   })
+  //     .then(res => res.json())
+  //     .then(res => {
+  //       this.setState({
+  //         commentList: res.data,
+  //         commentValue: ''
+  //       });
+  //     });
+  // }
+
+  render() {
+
+    const { newComment } = this.props;
+    return (
+
+      <div className="Comment" >
+        <ul>
+          {newComment.map((el) => {
+            return (
+              <li>{el.userName}  {el.content}</li>
```

Comment:
```suggestion
              <li>
                <span>{el.userName}</span>
                <span>{el.content}</span>
              </li>
```
- 위와 같이 내용 확실하게 분리하는 것도 좋고 나중에 스타일 다르게 적용해야 하는 경우에는 분리해주세요!

Reply:
대박, 스타일 다르게 적용해야하는데안되서 끙끙했는데, 이런간단한 방법이 있었네요,,,,,감사합니다!